### ISSUE TITLE
How about code in pytorch?

### ISSUE THREAD

**Issue by John1231983** [scut-aitcm/CompetitiveSENet#1]
Thanks for sharing your code. It is a great idea. Could you also implement your se-block in pytorch? I think many researchers included me are interested in it.

**Comment by luonango** [comment]
Thank you for your interests in our CompetitiveSENet work. 

I am very sorry that there is no pytorch code yet, because we are not familiar with pytorch.

We welcome everyone to implement CMPE-SENet in other frameworks. And if you have any questions about the implementation, please feel free to ask me about those puzzles. I will answer them with my best.

**Comment by John1231983** [comment]
It is fine. I will try to understand your propose and implement it in pytorch. One question about the layer after FC. Normally, the sigmoid is used to change the range to 0-1. Could you try softmax, instead of the sigmoid function? I think softmax is more meaningful than using sigmoid although the author's SeNet used it

**Comment by superhy** [comment]
Softmax causes antagonism between output labels, it is not what we want to happen, we just want to implement a new type of “channel-wise attention”. Referring to the most relevant study: **https://arxiv.org/abs/1709.01507** with ours, they also used Sigmoid.

**Comment by John1231983** [comment]
Thanks for your information. Could you explain what does "antagonism between output labels" mean?

**Comment by superhy** [comment]
In **http://ufldl.stanford.edu/wiki/index.php/Softmax_Regression**, see: "the classes are mutually exclusive, so a softmax regression classifier would be appropriate"
In our actual experiments, softmax usually results in one label's value being particularly large and the others being extremely small, thus destroying the SE effect.
